apps/prairielearn/src/pages/administratorWorkspaces/administratorWorkspaces.html.ts

```diff
@@ -161,17 +150,7 @@ function AdministratorWorkspaces({ workspaces, workspaceLoadHostCapacity, resLoc
   `.toString();
 }
 
-/**
- * @typedef {Object} CapacityProps
- * @property {number} total
- * @property {number} current
- */
-
-/**
- * @param {CapacityProps} props
- * @returns {import('@prairielearn/html').HtmlSafeString}
- */
-function Capacity({ total, current }) {
+function Capacity({ total, current }: { total: number; current: number }) {
```

Comment:
I noticed that the html.ts files in components include the return type (`HtmlSafeString`), while the files in pages don't. Should there be a convention in this case?